addons/im_livechat/static/tests/embed/chat_window.test.js

```diff
@@ -41,9 +41,10 @@ test("do not save fold state of temporary live chats", async () => {
     await waitNotifications([env, "discuss.Thread/fold_state"]);
     await contains(".o-mail-Message", { text: "Hello", count: 0 });
     await assertSteps(["fold - folded"]);
-    await contains(".o-mail-ChatBubble");
-    await click("[title*='Close Chat Window']");
+    await click(".o-mail-ChatBubble");
+    await click("[title='Close Chat Window']");
```

Comment:
Keep the `*=` as we should keep `(ESC)` part (but don't want to show in test because this is irrelevant)